taskgraph/Task.py

```diff
@@ -62,11 +57,11 @@ def daemon(self, value):
 
 
 # From https://stackoverflow.com/a/8963618/42897
-# "As the current implementation of multiprocessing [3.7+] has been extensively 
+# "As the current implementation of multiprocessing [3.7+] has been extensively
 # refactored to be based on contexts, we need to provide a NoDaemonContext
-# class that has our NoDaemonProcess as attribute. [NonDaemonicPool] will then 
-# use that context instead of the default one." 
-# "spawn" is chosen as default since that is the default and only context 

```

Comment:
Oops, these end line white spaces was probably me!

Reply:
Ahh if so, can you set your editor to chomp trailing whitespace on a save? If you're on SublimeText it's a matter of adding `"trim_trailing_white_space_on_save": true,` to your settings dictionary. I otherwise didn't notice I did this! The editor does it for me.